src/components/LinkDevice/blocks/Filters.tsx

```diff
@@ -15,7 +15,7 @@ type Props = {
 };
 
 const Filters: FC<Props> = ({ sites, siteId, isFetching, onChangeSearchQuery, onChangeSite }) => {
-  const options = useMemo(() => getOptions(sites, "companyName"), [sites]);
+  const options = useMemo(() => getOptions(sites, "brandingName"), [sites]) as Array<Option>;
```

Comment:
Can this be typed without using `as`? My guess is `getOptions` is missing some type information which is why it needs to be set it.

Reply:
I'm afraid not. Need to fix types in "app-sdk". I created story for it https://app.shortcut.com/deskpro/story/158765/app-sdk-need-to-fix-the-types-in-select-component

Reply:
`getOptions` is in this codebase, not the SDK, and so it `Option`. I'd expect `getOptions` to return `Option[]` so it doesn't have to be typed here.

Reply:
Could you help me with this type, please? How it must be